runtime/jailer_integ_test.go

```diff
@@ -79,3 +80,17 @@ func testJailer(t *testing.T, jailerConfig *proto.JailerConfig) {
 		require.NoError(err, "failed to delete a container")
 	}()
 }
+
+func TestJailerWithNumaNode_Isolated(t *testing.T) {
+	prepareIntegTest(t, withJailer())
+
+	t.Run("Jailer NumaNode", func(t *testing.T) {
+		b := cpuset.Builder{}
+		cset := b.AddCPU(0).AddMem(0).Build()
```

Comment:
This will work even on non-NUMA hardware, right?

Reply:
I cannot find an instance to test this on that has numa support disabled. I will try to test this on my local machine by disabling numa support all together in the BIOs.

Reply:
It should. I'm mainly asking because I wonder if we should give the test a more accurate name. Firecracker uses "NUMA" all over the place, even on non-NUMA systems, so it could get confusing anyway. Up to you...

Reply:
Renamed to `TestJailerCPUSet_Isolated`